tailwind.config.js

```diff
@@ -47,18 +55,18 @@ module.exports = {
     },
     fontSize: {
       // assets/styles/_type
-      "-3": "var(--calcite-font-size--3)",  // 10px
-      "-2": "var(--calcite-font-size--2)",  // 12px
-      "-1": "var(--calcite-font-size--1)",  // 14px
-      0: "var(--calcite-font-size-0)",      // 16px
-      1: "var(--calcite-font-size-1)",      // 18px
-      2: "var(--calcite-font-size-2)",      // 20px
-      3: "var(--calcite-font-size-3)",      // 26px
-      4: "var(--calcite-font-size-4)",      // 32px
-      5: "var(--calcite-font-size-5)",      // 40px
-      6: "var(--calcite-font-size-6)",      // 48px
-      7: "var(--calcite-font-size-7)",      // 56px
-      8: "var(--calcite-font-size-8)",      // 64px
+      "-3": [ "var(--calcite-font-size--3)", { lineHeight: '0.75rem' } ], // 10px (0.625rem)
+      "-2": [ "var(--calcite-font-size--2)", { lineHeight: '1rem' } ],    // 12px (0.75rem)
+      "-1": [ "var(--calcite-font-size--1)", { lineHeight: '1rem' } ],    // 14px (0.875rem)
+      "0": [ "var(--calcite-font-size-0)", { lineHeight: '1.25rem' } ],   // 16px (1rem)
+      "1": [ "var(--calcite-font-size-1)", { lineHeight: '1.5rem' } ],    // 18px (1.125rem)
+      "2": [ "var(--calcite-font-size-2)", { lineHeight: '1.5rem' } ],    // 20px (1.25rem)
+      "3": [ "var(--calcite-font-size-3)", { lineHeight: '2rem' } ],      // 26px (1.625rem)
+      "4": [ "var(--calcite-font-size-4)", { lineHeight: '2.5rem' } ],    // 32px (2rem)
+      "5": [ "var(--calcite-font-size-5)", { lineHeight: '3rem' } ],      // 40px (2.5rem)
+      "6": [ "var(--calcite-font-size-6)", { lineHeight: '4rem' } ],      // 48px (3rem)
+      "7": [ "var(--calcite-font-size-7)", { lineHeight: '4rem' } ],      // 56px (3.5rem)
+      "8": [ "var(--calcite-font-size-8)", { lineHeight: '5rem' } ],      // 64px (4rem)
```

Comment:
Including `line-height` inspired by the v2.x font size [utils](https://tailwindcss.com/docs/font-size). These are based on @bstifle 4px scale.